ControlFlow/Program.cs

```diff
@@ -48,7 +63,23 @@ public static string LandscapeOrPortrait(int width, int height)
         // Suspended" instead. Don't Worry about input validation.
         public static string SpeedTrap(int speedLimit, int speedCar)
         {
-            return default;
+            var SD = speedCar - speedLimit;
+            var DS = 0;
+            if (speedLimit >= speedCar)
+                return Convert.ToString("Okay");
+            else
+            {
+                for (int TF = 5;
+                    TF <= SD;
+                    TF += 1)
```

Comment:
This for loop should be all on one line, the way it is currently is very confusing to read.